common/src/main/java/ch/admin/bag/covidcertificate/common/config/CovidCertificateNewsItem.kt

```diff
@@ -0,0 +1,11 @@
+package ch.admin.bag.covidcertificate.common.config
+
+import com.squareup.moshi.JsonClass
+import java.io.Serializable
+
+@JsonClass(generateAdapter = true)
+data class CovidCertificateNewsItem(
+	val iconAndroid: String?,
+	val iconIos: String?,
```

Comment:
Since we don't use this icon anyway, it can be omitted in the response data class